h2o-py/h2o/display.py

```diff
@@ -79,6 +84,13 @@ def _in_ipy():  # are we in ipy? then pretty print tables with _repr_html
             except ImportError:
                 H2ODisplay._jupyter = False
         return H2ODisplay._jupyter
+        
+    @staticmethod
+    def _in_zep():  # are we in zeppelin? then use zeppelin pretty print support
+        if H2ODisplay._zeppelin is None:
+            import os
+            H2ODisplay._zeppelin = H2ODisplay._in_ipy() and "ZEPPELIN_RUNNER" in os.environ
```

Comment:
Probably extreme edge case, but what if someone has `ZEPPELIN_RUNNER` defined in their bash env for some obscure reasons?

Reply:
yes, that is unfortunately a possibility, however very unlikely, because this env var is set directly by ZEPPELIN and not by user/config
I am aware this can cause issues, but would like your oppinion on this

Reply:
I think it is fine to go this way, the case mentioned above is really rare I believe.

When I was initially looking into this issue, I was thinking of a solution which would print independently on Zeppelin/Jupyter, but I guess you have already try that.